docs/api/effector/babel-plugin.md

```diff
@@ -88,150 +88,151 @@ getUsers.use(
 
 ## Configuration
 
-### noDefaults
+### importName
+
+- Type: `string | string[]`
+- Default: `['effector', 'effector/compat']`
 
-> `noDefaults: boolean`
+Specifying import name or names to process by plugin. Import should be used in the code as specifed.
+
+### factories
 
 :::note since
-effector 20.2.0
+effector 21.6.0
 :::
 
-Option for effector/babel-plugin for making custom unit factories with clean configuration.
+- Type: `string[]`
+
+Accepts an array of module names which exports treat as custom factories therefore each function call provides unique prefix for [sids](./babel-plugin.md#sid) of units inside them. Used to SSR([Server Side Rendering](https://developers.google.com/web/updates/2019/02/rendering-on-the-web)) and it's not required for client-only application.
+
+- Factories can have any amount of arguments.
+- Factories can create any amount of units.
+- Factories can call any effector methods.
+- Factories can call another factories from others modules.
+- Modules with factories can export any amount of functions.
+- Factories should be compiled with `effector/babel-plugin` as well as code which use them.
+
+#### Example
 
 ```json title=".babelrc"
 {
   "plugins": [
-    ["effector/babel-plugin", {"addLoc": true}],
     [
       "effector/babel-plugin",
       {
-        "importName": "@lib/createInputField",
-        "storeCreators": ["createInputField"],
-        "noDefaults": true
-      },
-      "createInputField"
+        "factories": ["src/createEffectStatus", "~/createCommonPending"]
+      }
     ]
   ]
 }
 ```
 
-```js title="@lib/createInputField.js"
-import {createStore} from 'effector'
-import {resetForm} from './form'
+```js title="./src/createEffectStatus.js"
+import {rootDomain} from './rootDomain'
 
-export function createInputField(defaultState, {sid, name}) {
-  return createStore(defaultState, {sid, name}).reset(resetForm)
+export function createEffectStatus(fx) {
+  const $status = rootDomain
+    .createStore('init')
+    .on(fx.finally, (_, {status}) => status)
+  return $status
 }
 ```
 
-```js title="src/state.js"
-import {createInputField} from '@lib/createInputField'
-
-const foo = createInputField('-')
-/*
+```js title="./src/statuses.js"
+import {createEffectStatus} from './createEffectStatus'
```

Comment:
I thought, some bundlers can fail transformation of path here 🤔

The previous version with absolute paths was more stable.

Reply:
Do you have an example?